src/modules/organizations/containers/projects-container.jsx

```diff
@@ -42,7 +42,21 @@ class ProjectsContainer extends React.Component {
       const query = { sort: 'display_name', page };
 
       organization.get('projects', query).then((projects) => {
-        this.props.dispatch(setOrganizationProjects(projects));
+        const allProjects = organization.links.projects.map((projectID) => {
+          if (projects.filter(project => project.id === projectID).length) {
+            return projects.filter(project => project.id === projectID)[0];
```

Comment:
Is it possible to get allProjects without mapping over org.links.projects and running filter twice?

Reply:
yes that's ugly, I tried something with find, let me know you think